### DISCUSSION TITLE
How to increase file upload size limit for self-hosted Budibase

### DISCUSSION THREAD

**Discussion by daawaan4x** [Budibase/budibase#5457] [Help]
Hello, we have to store high-quality resolutions of scanned documents in pdf format. Unfortunately, the 20MB upload size limit might be too small for our use case. Is there a way to bypass this limit with environment variables or other methods? We're fine with building Budibase from source but there aren't any docs for deploying Budibase in production from a modified source unfortunately since all hosting configuration files point to the official images built by Budibase in the cloud

**Comment by ghost** [comment]
@daawaan May i know the 20 MB upload size limit is defined from budibase or from the web server you are using like nginx ?

**Comment by philsonchoi** [comment]
I'm having this same issue. Does anyone have a workaround for allowing file uploads up to 20 GBs via AWS S3?

**Comment by mlucero-wine** [comment]
Any update on this? Same issue here, running budibase in Kubernetes. I updated these variables but no change:
``` 
MAX_IMPORT_SIZE_MB:         2000
HTTP_MB_LIMIT:              2000
```
